pcstac/pcstac/client.py

```diff
@@ -45,18 +45,6 @@ class PCClient(CoreCrudClient):
 
     extra_conformance_classes: List[str] = attr.ib(factory=list)
 
-    def conformance_classes(self) -> List[str]:
-        """Generate conformance classes list."""
-        base_conformance_classes = self.base_conformance_classes.copy()
-
-        for extension in self.extensions:
-            extension_classes = getattr(extension, "conformance_classes", [])
-            base_conformance_classes.extend(extension_classes)
-
-        base_conformance_classes.extend(self.extra_conformance_classes)
-
-        return sorted(list(set(base_conformance_classes)))
```

Comment:
this was the same as https://github.com/stac-utils/stac-fastapi/blob/80064f77739ddc1acaafa78cb0d8a11f77ecd33e/stac_fastapi/types/stac_fastapi/types/core.py#L348-L357